### PR TITLE
Fixes #2481 NTR- choroid plexus capillary endothelial cell 

### LINKED ISSUE #2481: [NTR] choroid plexus capillary endothelial cell
Please check that the term does not already exist by using the ontology search tool OLS:
https://www.ebi.ac.uk/ols4/ontologies/cl

**Preferred term label**
choroid plexus capillary endothelial cell

**Synonyms** (add reference(s), please)

**Definition** (free text, with reference(s), please. PubMed ID format is PMID:XXXXXX)
A capillary endothelial cell that is part of the choroid plexus storm, characterized by its fenestrated nature with 60 to 80 nm fenestrations and lack of tight junctions. This fenestrated structure allows for the rapid delivery of water and other components, aiding in the production of cerebrospinal fluid (CSF). 
PMID: [18480313](https://pubmed.ncbi.nlm.nih.gov/18480313)
PMID:32375819
https://www.ncbi.nlm.nih.gov/books/NBK27998/

**Parent cell type term** (check the hierarchy here https://www.ebi.ac.uk/ols4/ontologies/cl)
[fenestrated endothelial cell](http://purl.obolibrary.org/obo/CL_0000666)
capillary endothelial cell

**Anatomical structure where the cell type is found** (check Uberon for anatomical structures: https://www.ebi.ac.uk/ols4/ontologies/uberon)
choroid plexus stroma
endothelium of choroid plexus capillary (new term to be added to uberon)
**Your ORCID**

**Additional notes or concerns**

### PR DESCRIPTION
**Pull request by Caroline-99**
Fixes #2481 

Added a new term with logical and textual defs and references

### CODE REVIEW

**Comment by Caroline-99** [comment]
#gogoeditdiff

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -32727,6 +32728,15 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:7107724") oboInOwl:hasRe
 AnnotationAssertion(rdfs:label obo:CL_4052011 "follicle associated enterocyte")
 SubClassOf(obo:CL_4052011 obo:CL_0000584)
 
+# Class: obo:CL_4052017 (choroid plexus capillary endothelial cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:18480313") Annotation(oboInOwl:hasDbXref "PMID:32375819") Annotation(oboInOwl:hasDbXref "https://www.ncbi.nlm.nih.gov/books/NBK27998/") obo:IAO_0000115 obo:CL_4052017 "A capillary endothelial cell that is part of the choroid plexus stroma, characterized by its fenestrated nature with 60 to 80 nm fenestrations and lack of tight junctions. This fenestrated structure allows for the rapid delivery of water and other components, aiding in the production of cerebrospinal fluid (CSF).")
+AnnotationAssertion(terms:contributor obo:CL_4052017 <https://orcid.org/0009-0000-8480-9277>)
+AnnotationAssertion(terms:date obo:CL_4052017 "2024-10-01T11:38:22Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4052017 "choroid plexus capillary endothelial cell")
+EquivalentClasses(obo:CL_4052017 ObjectIntersectionOf(obo:CL_0002144 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005206)))
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:32737]: should be part of 'choroid plexus', not 'choroid plexus stroma' which is defined as "Stromal matrix _surrounding_ blood vessels within the choroid plexus"

**Review by dosumis — CHANGES_REQUESTED** [review]

**Comment by Caroline-99** [comment]
#gogoeditdiff

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -32791,6 +32792,15 @@ SubClassOf(obo:CL_4052019 obo:CL_0000066)
 SubClassOf(obo:CL_4052019 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0003889))
 SubClassOf(obo:CL_4052019 ObjectSomeValuesFrom(obo:CL_4030045 obo:GO_0005929))
 
+# Class: obo:CL_4052017 (choroid plexus capillary endothelial cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:18480313") Annotation(oboInOwl:hasDbXref "PMID:32375819") Annotation(oboInOwl:hasDbXref "https://www.ncbi.nlm.nih.gov/books/NBK27998/") obo:IAO_0000115 obo:CL_4052017 "A capillary endothelial cell that is part of the choroid plexus stroma, characterized by its fenestrated nature with 60 to 80 nm fenestrations and lack of tight junctions. This fenestrated structure allows for the rapid delivery of water and other components, aiding in the production of cerebrospinal fluid (CSF).")
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:32797]: Text def still says choroid plexus stroma.  Otherwise now good.

**Review by dosumis — APPROVED** [review]

**Review by dosumis — APPROVED** [review]

**Pull request merged**